testing/helper.test.js

```diff
@@ -0,0 +1,67 @@
+import * as helperFunc from "../javascript/helper.js";
+
+const { convertToMorse, isInList, convertToEnglish, invalidChar, emptySpace } = helperFunc;
+
+describe("isInList() test - checks if a given value exists in the morse code array", () => {
+	it("should return the alphabet or symbol when valid morse code is passed", () => {
+		expect(isInList("....")).toBe("H");
+		expect(isInList(".____")).toBe("1");
+		expect(isInList("_.._.")).toBe("/");
+	});
+	it("should return undefined when invalid value is passed", () => {
+		expect(isInList("ac")).toBe(undefined);
+		expect(isInList("33")).toBe(undefined);
+		expect(isInList(true)).toBe(undefined);
+		expect(isInList("--...")).toBe(undefined);
+	});
+});
+describe("convertToMorse() - converts english to morse code", () => {
+	it("should return string value", () => {
+		expect(typeof convertToMorse("hello")).toBe("string");
+		expect(typeof convertToMorse("Please help me!")).toBe("string");
+		expect(typeof convertToMorse("19897856")).toBe("string");
+	});
+	it("should return correct morse code", () => {
+		expect(convertToMorse("hello")).toBe(".... . .-.. .-.. ---");
+		expect(convertToMorse("I'm 99 years")).toBe(`.. .____. -- ${emptySpace} ____. ____. ${emptySpace} -.-- . .- .-. ...`);
```

Comment:
I will look into an alternative way to indicate the spacing for the user as the empty space isn't prominent. Thanks.

Reply:
I have decided to use "/" to indicate the spacing when translating from English to Morse. On the other hand, I have decided to keep spacing as it was when translating Morse to English as it can be human-readable when it outputs the translation results.
I have also adjusted the testing based on the changes made.